### ISSUE TITLE
Duplicate attribute

### ISSUE THREAD

**Issue by lu7766lu7766** [johnsoncodehk/volar#665]
It just happened recently.
I have used pug and tailwind css, and used vetur to format the code
Is it a bug? or how can I disabled alert
I have add .eslintrc file and set , but it's not work
```
{
  "vue/no-duplicate-attributes": ["error", {
    "allowCoexistClass": true,
    "allowCoexistStyle": true
  }]
}
```

![image](https://user-images.githubusercontent.com/13795053/140337281-37fcb071-fee1-450f-bcc6-8cb8cae2a015.png)

**Comment by johnsoncodehk** [comment]
It should be a regression of 0.28.10, you can use 0.28.9 for now.

**Comment by johnsoncodehk** [comment]
Sorry I just found that this is a new limitation due to resolved #618.

You can't mixed use `.aaa(class="bbb" ...)`, please change to `.aaa.bbb(...)` or `div(class"aaa bbb")`.
